.env.example

```diff
@@ -6,5 +6,5 @@ PGDATABASE=hurumap
 PGUSER=hurumap
 PGPASSWORD=hurumap
 PYTHONDONTWRITEBYTECODE="True"
-USE_MAPIT="True"
+USE_MAPIT=True
```

Comment:
Change `PYTHONDONTWRITEBYTECODE="True"` to `PYTHONDONTWRITEBYTECODE=True` as well.